README.md

```diff
@@ -44,3 +44,10 @@ yarn start
 ```bash
 bash src/provers/light-optimistic/deploy-heroku.sh <heroku-app-name>
 ```
+
+### Deploy to Docker
+
+```bash
+docker run -p 8546:8546 --name kevlar shresthagrawal/kevlar
```

Comment:
@shresthagrawal Let's make sure to update this in case your `DOCKERHUB_USERNAME` isn't `shresthagrawal`.

Reply:
The username is correct